tests/common/reboot.py

```diff
@@ -223,12 +224,27 @@ def check_reboot_cause(dut, reboot_cause_expected):
     logging.debug("dut {} last reboot-cause {}".format(dut.hostname, reboot_cause_got))
     return reboot_cause_got == reboot_cause_expected
 
+
 def sync_reboot_history_queue_with_dut(dut):
     """
     @summary: Sync DUT and internal history queues
     @param dut: The AnsibleHost object of DUT.
     """
 
+    global REBOOT_TYPE_HISTOYR_QUEUE
+    global MAX_NUM_REBOOT_CAUSE_HISTORY
+
+    # Initialize local deque for storing DUT reboot cause history
+    dut_reboot_history_queue = deque([], MAX_NUM_REBOOT_CAUSE_HISTORY)
+
+    # Skip this function if sonic image is 201811 or 201911
+    if "201811" in dut.os_version or "201911" in dut.os_version:
```

Comment:
Hi,
there is a well-defined api too: 

from tests.common.utilities import skip_release
  skip_release(duthost, ["201811", "201911"])

Reply:
Hi Wenyi,

I decided against using skip_release because that API skips the entire test. I need only the sync function to be skipped.